### ISSUE TITLE
RfBuilder shape card Shows no shape in GUI

### ISSUE THREAD

**Issue by Volensq** [NillerMedDild/Enigmatica6#458]
Enigmatica6-0.3.0
Forge 35.1.0

RFBuilder/ RFTools.,  The builder block and corresponding shape/Quarry items Should have a GUI in them showing the shape being outlined.. They do not. it is blank.
As well, the Spawner block from RFBuilder, that infuses mod DNA with a syringe will sometimes not show the items needed to spawn the mob/ -What to infuse like Netherrock/ ect for example
Singleplayer/ and Multiplayer.

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by NillerMedDild** [comment]
I relayed it :)

**Comment by Gayanfalor** [comment]
Hello, talking about the builder, is it working for you ? i'm in e6 ver 0.3.2 i tried to make one wokring i specified the area it should work but its starting to get dirt etc and stop, it had chest over it energy and allowed in ally with ftb team

**Comment by NillerMedDild** [comment]
@Gayanfalor Works just fine for me. Did you give it a redstone signal or change the redstone setting?

**Comment by Gayanfalor** [comment]
yeah i put a redstone singal to turn it on dunno why its not working as it should
